content/en/docs/quick-start/enable-pluggable-components.md

```diff
@@ -36,7 +36,7 @@ For more information about each component, see Overview of Enable Pluggable Comp
 
 ### **Installing on Linux**
 
-When you install KubeSphere on Linux, you need to create a configuration file, which lists all KubeSphere components.
+When you install KubeSphere on multi-node Linux, you need to create a configuration file, which lists all KubeSphere components.
```

Comment:
->
When you implement multi-node installation of KubeSphere on Linux, you need to create a configuration file, which lists all KubeSphere components.